starlight/static/css/main.css

```diff
@@ -0,0 +1,28 @@
+@import url('https://fonts.googleapis.com/css?family=Proxima+Nova');
+
+body {
+  /*background: url('/static/images/starlight-background.png');*/
+  background-color: lightgray;
+  background-size: 100% auto;
+  font-family: 'proxima-nova';
+}
+body > .grid {
+  height: 100%;
+}
+.image {
+  margin-top: -100px;
+}
+.competency-table {
+  margin-left:50px;
+  margin-right:50px;
+  vertical-align:right;
+  overflow-x:scroll;
+}
+.ui secondary menu{
+  color: red
+}
+
+.login-form{
```

Comment:
If you'd like, you can use the grid system in Semantic UI to more cleanly / responsively size your forms, tables and elements. Check it out: https://semantic-ui.com/collections/grid.html